xmppserver/pom.xml

```diff
@@ -462,15 +462,33 @@
         </dependency>
         <!-- Test Scope -->
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.13.1</version>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
I'd be very tempted to use the JUnit BOM for this. 
```
        <dependency>
            <groupId>org.junit</groupId>
            <artifactId>junit-bom</artifactId>
            <version>5.9.3</version>
            <type>pom</type>
            <scope>import</scope>
        </dependency>
```
This removes the need to set the version for any of your JUnit dependencies - so next time you want to update the version, change it in the one place.